websites/F/FRAnime/presence.ts

```diff
@@ -0,0 +1,132 @@
+// Init the client
+const presence = new Presence({
+		clientId: "1102272266278027296",
+	}),
+	// Define playerAssets
+	playerAssets = {
+		logo: "https://i.imgur.com/Fe6SuHW.png",
+		logoBg: "https://i.imgur.com/byFtXWd.png",
+		pause: "https://i.imgur.com/0A75vqT.png",
+		play: "https://i.imgur.com/Dj5dekr.png",
+		search: "https://i.imgur.com/C3CetGw.png",
```

Comment:
Use standardized assets. It may be easier to put existing assets in an enum like so:
https://github.com/PreMiD/Presences/blob/1f5a4117f92c6549a75e1c01c6305f9c4398b5b5/websites/A/Aodesu/presence.ts#L6-L14

See the index.d.ts file for existing standard Assets you can use.

Reply:
This has not been resolved yet. Please put your assets in an `Assets` enum, and make use of the standard ones provided.